plugins/woocommerce/src/Internal/Email/OrderPriceFormatter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * OrderPriceFormatter class file.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\Email;
+
+use WC_Abstract_Order;
+
+/**
+ * Helper class for formatting prices in order emails.
+ *
+ * @internal Just for internal use.
+ */
+class OrderPriceFormatter {
+
+	/**
+	 * Gets item subtotal - formatted for display in emails.
+	 *
+	 * @param WC_Abstract_Order $order Order instance.
+	 * @param object            $item Item to get unit price from.
+	 * @param string            $tax_display Incl or excl tax display mode.
+	 * @return string Formatted item subtotal.
+	 */
+	public static function get_formatted_item_subtotal( WC_Abstract_Order $order, $item, string $tax_display = '' ): string {
+		$tax_display   = $tax_display ? $tax_display : get_option( 'woocommerce_tax_display_cart' );
+		$include_tax   = 'excl' !== $tax_display;
+		$item_subtotal = $order->get_item_subtotal( $item, $include_tax );
+		return self::format_price( $order, $item_subtotal, $include_tax );
+	}
+
+	/**
+	 * Helper method to format price with or without tax.
+	 *
+	 * @param WC_Abstract_Order $order Order instance.
+	 * @param float             $amount The amount to format.
+	 * @param bool              $include_tax Whether to include tax in the formatted price.
+	 * @return string Formatted price string.
+	 */
+	private static function format_price( WC_Abstract_Order $order, float $amount, bool $include_tax ): string {
+		if ( $include_tax ) {
```

Comment:
Can be simplified a bit:

```php
'ex_tax_label' => ( $include_tax && $order->get_prices_include_tax() ) ? 1 : 0,
```

Reply:
Yup, `ex_tax_label` is deafult to be false/0. Updated in https://github.com/woocommerce/woocommerce/pull/57573/commits/eada008cd7681ce8821141bc4ce854625b864141.

Just for reference, the price display behavior should match the pre-existing `order.get_formatted_line_subtotal`:

https://github.com/woocommerce/woocommerce/blob/8400fe522772229b3b6193c93f28fccdc7258d33/plugins/woocommerce/includes/abstracts/abstract-wc-order.php#L2158-L2173

Ideally, both formatted subtotal functions can share the price display logic. But there are use cases of `get_formatted_line_subtotal` outside of email templates and I didn't want to introduce risk of regression.